app/assets/javascripts/app/controllers/_ui_element/select_organization.coffee

```diff
@@ -0,0 +1,5 @@
+class App.UiElement.select_organization extends App.UiElement.select
+  @render: (attribute, params) ->
+    if attribute['name'] == 'organization_id' && attribute['default'] == 0
```

Comment:
Why is there a check for the attribute name if the name of the class is already limited to the attribute?